sdk/storage/azure-storage-queue/azure/storage/queue/_encryption.py

```diff
@@ -491,6 +491,7 @@ def _generate_encryption_data_dict(
     :rtype: Dict[str, Any]
     """
     # Encrypt the cek.
+    wrapped_cek = bytes()
```

Comment:
We should instead raise an exception in an `else` that you can add to the below if. There should be a few examples in this file of the exact exception to raise for invalid version.